MidasFrontend/ArduinoHelper.cxx

```diff
@@ -0,0 +1,195 @@
+#include "ArduinoHelper.h"
+#include "TestStandMessages.h"
+#include "Gantry.h"
+
+#include "LinuxSerialDevice.h"
+#include "TestStandCommHost.h"
+
+#include "shared_defs.h"
+
+#include "midas.h"
+
+#include <stdio.h>
+#include <math.h>
+
+#define BAUD_RATE           115200
+#define MSG_RECEIVE_TIMEOUT 5000
+
+LinuxSerialDevice device;
+TestStandCommHost comm(device);
+
+// Gantry Mechanical Properties
+const float pulley_diameter_mm   = 17;
+const float dist_per_rev_mm      = 2 * M_PI * (pulley_diameter_mm / 2);
```

Comment:
I'm still unsure whether having integers like `2` would cause truncations in the calculation and then subsequently get casted into a `float` since the datatype is float. Would it be safer to add decimal places. ie `2.0 * M_PI * (pulley_diameter_mm / 2.0)`

Reply:
If at least one of the operands is a float then the result will be a float, so we're safe here. That being said, doesn't hurt to add the `.0` so I can go ahead and add it

Reply:
Just an architectural? question - what's the general rule for using `#define  CONSTANT 7` vs `const int = 7;`

Reply:
This is a fun one.
`#define MAGIC 7` (in this case) comes from me forgetting that I'm writing C++ and not C. In C, `const int magic = 7;` is not actually a "constant expression" which means there's a whole bunch of scenarios where you can't use it like a constant (e.g. case labels, array dimensions). Essentially, `const` variables in C are not truly constant, so the general practice is to use `#define`s or `enum`s instead which are truly constant.
In C++, `const` variables are truly constant (well sometimes you have to declare them `constexpr`), so there's actually very few scenarios if any where you would use a `#define`.

So given that MIDAS is now C++ ish sorta kinda, I should probably swap out my defines for consts.

Reply:
These constants ended up moving in the next PR so I will switch them from defines later